index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
 
 <script>
   // Initialization
```

Comment:
```suggestion
  // Initialization
``` Maybe have a weekly drop in session where students can ask questions and you can check their progress so far,